hybrisprovider.cpp

```diff
@@ -1205,6 +1213,22 @@ void HybrisProvider::technologiesChanged()
     } else {
         qCDebug(lcGeoclueHybris) << "Cellular technology not available";
     }
+
+    if (m_wifiTechnology) {
+        disconnect(m_wifiTechnology, SIGNAL(connectedChanged(bool)),
+                   this, SLOT(wifiConnected(bool)));
+    }
+
+    m_wifiTechnology = m_networkManager->getTechnology(QStringLiteral("wifi"));
+
+    if (m_wifiTechnology) {
+        qCDebug(lcGeoclueHybris) << "Wifi technology available and"
+                                 << (m_wifiTechnology->connected() ? "connected" : "not connected");
+        connect(m_wifiTechnology, SIGNAL(connectedChanged(bool)),
+                this, SLOT(wifiConnected(bool)));
+    } else {
+        qCDebug(lcGeoclueHybris) << "Wifi technology not available";
+    }
```

Comment:
I though that there is some higher level API that just returns data connection or not? From our point of view we shouldn't care if it is cellular or wifi should we?